File: football-backend/src/main/java/com/testowanie/football/model/Editor.java
package com.testowanie.football.model;

import jakarta.persistence.*;
import lombok.*;
import org.springframework.lang.NonNull;

@Entity
@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class Editor {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @NonNull
    @Column(nullable = false)
    private String name;

    @NonNull
    @Column(nullable = false)
    private String surname;

    private String photoUrl;
}

File: football-frontend/src/app/models/editor.ts
export class Editor {
  id?: number;
  name: string;
  surname: string;
  photoUrl: string;

  constructor(name: string, surname: string, photoUrl: string, id?: number) {
    this.name = name;
    this.surname = surname;
    this.photoUrl = photoUrl;
    this.id = id;
  }
}
